support-frontend/assets/pages/[countryGroupId]/thankYou.tsx

```diff
@@ -104,15 +104,11 @@ export function ThankYou({ geoId, appConfig }: ThankYouProps) {
 
 			const getFulfilmentOptions = (productKey: string): FulfilmentOptions => {
 				switch (productKey) {
-					case 'SupporterPlus':
-					case 'Contribution':
-						return 'NoFulfilmentOptions';
 					case 'TierThree':
 						return countryGroupId === 'International'
 							? 'RestOfWorld'
 							: 'Domestic';
 					default:
-						// ToDo: define for every product here
```

Comment:
If we remove this ToDo could it bite us when we do the newspaper products? I'm sure we'd figure it out then tbh!

Reply:
Agree, I think we can figure this out when new delivery cases need adding. Will leave as-is, hope thats ok.